### DISCUSSION TITLE
Pre-release links are broken(return 404)

### DISCUSSION THREAD

**Discussion by zoulja** [sumatrapdfreader/sumatrapdf#2375] [General]
Subj

**Comment by GitHubRulesOK** [comment]
@KJK the redirection needs update 
![image](https://user-images.githubusercontent.com/12754913/155737082-3169ef9c-6271-4fb0-9e85-13b3987dfaa2.png)

[later edit] outdated links removed
is currently at ...
however prerel pages says ...

**Comment by kjk** [comment]
Fixed with https://github.com/sumatrapdfreader/sumatrapdf/commit/9228277f23c5223c703d2460b57509b10cbf8eb1
